packages/clay-css/src/scss/functions/_lx-icons-generated.scss

```diff
@@ -814,7 +814,7 @@
 
 	'shield-check': '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path class="lexicon-icon-outline" d="M474.7 54.4c-3.5-5.5-8.6-10-14.6-12.7C395.8 13.6 326.2-.6 256 0 185.8-.6 116.2 13.6 51.9 41.6c-6 2.8-11.1 7.2-14.6 12.7-3.5 5.5-5.4 12-5.3 18.6v170c-.1 55.2 16.6 109.1 47.9 154.6 31.3 45.6 75.7 80.6 127.4 100.7l23.9 9.2c15.9 6.1 33.6 6.1 49.5 0l23.9-9.2c51.7-20 96.2-55.1 127.4-100.7 31.3-45.6 48-99.5 47.9-154.6v-170c.1-6.5-1.7-13-5.2-18.5zM419 170.3 251.7 353.7c-5.1 5.4-12.2 8.6-19.6 8.8h-.8c-7.2 0-14.2-2.7-19.4-7.7l-96.5-92.5c-6.3-6.3-8.6-13.4-8-20.1.7-6.7 4.2-12.6 9.1-17 9.7-8.7 26-11.9 37.7-.2l75.8 72.7 148-162.5c5.8-6.3 12.9-8.9 19.9-8.8 6.9.1 13.4 2.9 18.4 7.2 5 4.3 8.7 10.2 9.6 16.8.9 6.6-1.1 13.6-6.9 19.9z" fill="#{$color}"/></svg>',
 
-	'shopping-cart': '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path class="lexicon-icon-outline lx-shopping-cart-basket" d="M93.2 51.2 181 255.9h212.1l24.7-124.2c10.5-43.1 72.1-28.5 60.8 20.2l-31.8 145.3c-4 13.1-16.2 22.7-30.6 22.7H159.9c-13.4 0-24.9-8.2-29.7-19.9L34.3 76.5c-14-36.7 38.9-64.9 58.9-25.3z" fill="#{$color}"/><ellipse class="lexicon-icon-outline lx-shopping-cart-wheel-1" cx="143.9" cy="432" rx="48.1" ry="48" fill="#{$color}"/><ellipse class="lexicon-icon-outline lx-shopping-cart-wheel-2" cx="400.2" cy="432" rx="48.1" ry="48" fill="#{$color}"/></svg>',
+	'shopping-cart': '<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path class="lexicon-icon-outline" d="M6.502 12.002a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm6 0a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Z" fill="#{$color}"/><path class="lexicon-icon-outline" fill-rule="evenodd" d="M15.102 3.01c.656.093 1.08.856.825 1.56l-2.067 5.713c-.137.38-.445.645-.802.705l-.156.014H5.62l-.155-.014a1.037 1.037 0 0 1-.735-.549l-.068-.156-2.16-7.281h-1.5a1 1 0 0 1 0-2h2a2 2 0 0 1 2 2h9.967l.133.009ZM6.31 9.002h5.922l1.27-4h-8.5l1.308 4Z" clip-rule="evenodd" fill="#{$color}"/></svg>',
```

Comment:
@pat270 The viewbox is switching from 512 to 16 here. Is that intended?

Reply:
@ethib137 yes, we have been converting the viewBox to 16 because the icons in Figma are 16. I can't seem to find the ticket, but @drakonux could fill you in more. I haven't seen any issues of things breaking due to this change.